templates/english/topic-page.html

```diff
@@ -0,0 +1,475 @@
+                  Heading link
+                </gcds-link>
+              </gcds-heading>
+              <gcds-text>
+                Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+              </gcds-text>
+            </div>
+            <div>
+              <gcds-heading
+                margin-bottom="100"
+                margin-top="0"
+                tag="h3"
+              >
+                <gcds-link
+                  href="#"
+                  class="font-text"
+                >
+                  Heading link
+                </gcds-link>
+              </gcds-heading>
+              <gcds-text>
+                Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+              </gcds-text>
+            </div>
+          </gcds-grid>
+        </section>
+
+        <!---------- Featured ---------->
+        <!-- TODO: Update the heading, image and text content. -->
+        <section>
+          <gcds-sr-only>
+            <gcds-heading margin-top="0" margin-bottom="0" tag="h2">Featured</gcds-heading>
+          </gcds-sr-only>
+          <gcds-grid
+            tag="div"
+            columns-desktop="2fr 1fr"
+            columns-tablet="1fr"
+            class="mt-400"
+          >
+            <gcds-grid
+              tag="div"
+              columns-desktop="1fr 1fr"
+              columns-tablet="1fr 1fr"
+            >
+              <img src="https://dummyimage.com/360x203/000000/FFFFFF.png" alt="" class="img-full">
+              <div>
+                <gcds-heading
+                  margin-bottom="200"
+                  margin-top="0"
+                  tag="h3"
+                >
+                  <gcds-link
+                    href="#"
+                    class="font-text"
+                  >
+                    Heading link
+                  </gcds-link>
+                </gcds-heading>
+                <gcds-text>
+                  Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+                </gcds-text>
+              </div>
+            </gcds-grid>
+            <div>
+              <gcds-heading
+                margin-top="0"
+                tag="h2"
+              >
+                <span class="font-h6">
+                  On social media
+                </span>
+              </gcds-heading>
+              <ul class="ms-400 d-block social-media">
+                <li class="mb-100">
+                  <gcds-link href="#" class="font-bold facebook">
+                      <gcds-sr-only>
+                        Facebook:
+                      </gcds-sr-only>
+                      Facebook
+                  </gcds-link>
+                </li>
+                <li class="mb-100">
+                  <gcds-link href="#" class="font-bold twitter">
```

Comment:
Could we use the `<strong>` tag instead of adding the `font-bold` class? A screen reader conveys emphasis when it encounters text within the <strong> tag which it won't do for the CSS bold styling.